examples/integrations.ipynb

```diff
@@ -0,0 +1,842 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Integrations with other tools"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import matplotlib.pyplot as plt\n",
+    "import mdtraj as md\n",
+    "traj = md.load(\"5550217/kras.xtc\", top=\"5550217/kras.pdb\")\n",
+    "topology = traj.topology"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from contact_map import ContactFrequency"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "traj_contacts = ContactFrequency(traj)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Exporting contact data\n",
+    "\n",
+    "Contact Map Explorer makes it easy for you to get your contact data into other formats, so that you can use the tools you're already familiar with to perform your analysis. Note that all of these report as integers, so what you have is the index of the object in the MDTraj topology,"
```

Comment:
what does "these"  mean in this sentence?
is it the contacts?:
```suggestion
    "Contact Map Explorer makes it easy for you to get your contact data into other formats, so that you can use the tools you're already familiar with to perform your analysis. Note that all of the contacts report as a pair of integers, so what you have is the index of the object in the MDTraj topology,"
```
or the residues/atoms
```suggestion
    "Contact Map Explorer makes it easy for you to get your contact data into other formats, so that you can use the tools you're already familiar with to perform your analysis. Note that the residues and atoms of these report as integers, so what you have is the index of the object in the MDTraj topology,"
```